src/sidebar-counters/sidebar-counters.js

```diff
@@ -4,66 +4,80 @@
 import { h } from 'dom-chef'
 import elementReady from 'element-ready'
 import api from '../api'
+import SelectorObserver from 'selector-observer'
 
 import './sidebar-counters.css'
 
-export const addBadge = (
-    a: HTMLAnchorElement,
-    resources: {| size: number |} | void
-) => {
-    const navLinksContainer: HTMLElement = (a.parentNode: any)
-    navLinksContainer.style.overflow = 'hidden'
+const HREF_BRANCHES = 'branches'
+const HREF_PULL_REQUESTS = 'pull-requests'
+export const menus = [HREF_BRANCHES, HREF_PULL_REQUESTS]
 
-    a.style.position = 'relative'
+const MAX_COUNTER = 99
 
-    const size =
-        // eslint-disable-next-line eqeqeq, no-eq-null
-        resources == null || resources.size == null
-            ? '?'
-            : resources.size > 999999
-                ? '999999+'
-                : resources.size
+export let menusCounters = {
+    HREF_BRANCHES: null,
+    HREF_PULL_REQUESTS: null,
+}
```

Comment:
Does this have to be a global variable in the file? Can we pass this as an argument to the function that needs it instead ?

That would also make it impossible to expose it to the public through an `export` and prevent the unit tests from relying on implementation details to assert functionality. We should be able to test this feature without having to manipulate internal state of the implementation